gallery/gallery/lib/l10n/gallery_localizations.dart

```diff
@@ -1812,6 +1812,61 @@ class GalleryLocalizations {
         desc: r'Title for the simple dialog component demo.');
   }
 
+  String demoSnackbarsAction(Object index) {
+    return Intl.message(r'You pressed snackbar $index’s action.',
+        locale: _localeName,
+        name: 'demoSnackbarsAction',
+        desc:
+            r'Text that appears when you press on a snackbars action. Make sure to not translate index.',
+        args: <Object>[index]);
+  }
+
+  String get demoSnackbarsActionButtonLabel {
+    return Intl.message(r'ACTION',
+        locale: _localeName,
+        name: 'demoSnackbarsActionButtonLabel',
+        desc: r'Label for action button text on the snackbar.');
+  }
+
+  String get demoSnackbarsButtonLabel {
+    return Intl.message(r'SHOW A SNACKBAR',
+        locale: _localeName,
+        name: 'demoSnackbarsButtonLabel',
+        desc: r'Label for button to show a snackbar.');
+  }
+
+  String get demoSnackbarsDescription {
+    return Intl.message(
+        r'Snackbars inform users of a process that an app has performed or will perform. They appear temporarily, towards the bottom of the screen. They shouldn’t interrupt the user experience, and they don’t require user input to disappear.',
+        locale: _localeName,
+        name: 'demoSnackbarsDescription',
+        desc: r'Description for snackbars demo.');
+  }
+
+  String get demoSnackbarsSubtitle {
+    return Intl.message(
+        r'Snackbars provide brief messages about app processes at the bottom of the screen.',
```

Comment:
Remove the period, and maybe try to shorten this if possible

Reply:
Will modify it to: "Snackbars show messages at the bottom of the screen"